libs/books/feature/src/lib/book-search/book-grid/book-grid-item/book-grid-item.component.ts

```diff
@@ -12,14 +10,28 @@ import { Book } from '@tmo/shared/models';
   changeDetection: ChangeDetectionStrategy.OnPush
 })
 
-export class BookGridItemComponent {
+export class BookGridItemComponent implements OnInit {
 
   @Input() book: ReadingListBook;
+  isFinished: boolean;
 
   constructor(
     private readonly store: Store
   ) {}
 
+  ngOnInit() {
+    this.setIsFinished(this.book);
+  }
+
+  setIsFinished(book) {
+    this.store.select(getReadingList).subscribe(
```

Comment:
What is going on here? Shouldn't this be dispatching an action?

Reply:
I am removing this because the input `book` will now have the `finished` property we needed. is there an action that should be dispatched here?